### DISCUSSION TITLE
200 page 오탈자 : reject -> catch

### DISCUSSION THREAD

**Discussion by nkpower0** [velopert/dealing-with-react-native#34] [버그]
책 200페이지 중간 예제 바로 윗 문단에...

> Promise에서 이행(resolved)된 결과는 Promise 개체의 then 메서드를 통해 사용할 수 있고, 거부(reject)된 오류는 ~~reject~~ catch 메서드를 통해 사용할 수 있습니다.

와 같이 **catch** 블럭으로 수정되어야 할 것 같습니다.

**Comment by velopert** [comment]
감사합니다! 
다음 중쇄때 반영하겠습니다.
